web_switch_company_warning/static/src/js/switch_company_warning_worker.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+//Show a big banner in the top of the page if the company has been
+//changed in another tab or window (in the same browser)
```

Comment:
[W7903(javascript-lint), ] Expected space or tab after '//' in comment. [Error/spaced-comment]